### PR TITLE
Remove init func

### PR DESCRIPTION
**Pull request by sebrandon1**
Trying to add unit tests to the QE repo to solve a problem and it is trying to run the `init()` function which is populating a list of clients which is not needed during unit test time.  

Bonus:
- Fixed the `make unit-tests` to work with this change.
- Modified the yamllint linter config based on the `cnf-certification-test` repo.
- Check out the `cnf-certification-test` repo because we need the TNF_REPO_PATH var set for the unit tests.

### CODE REVIEW

File: tests/globalhelper/init.go [hunk 1]
```diff
@@ -2,35 +2,26 @@ package globalhelper
...
 	if err != nil {
 		glog.Fatal(fmt.Sprintf("can not load api client. Please check KUBECONFIG env var - %s", err))
 	}
 
-	Configuration, err = config.NewConfig()
+	return c
+}
 
+func NewConfiguration() *config.Config {
+	Configuration, err := config.NewConfig()
```
Comment (edcdavid) [inline tests/globalhelper/init.go:21]: Don't we need to follow a singleton pattern here? looks like the configuration is re-created everytime

File: tests/globalhelper/init.go [hunk 2]
```diff
@@ -2,35 +2,26 @@ package globalhelper
...
-		Configuration = &config.Config{}
-
-		return
-	}
-
-	var err error
-	APIClient, err = config.DefineClients()
-
+func NewAPIClient() *testclient.ClientSet {
+	c, err := config.DefineClients()
```
Comment (edcdavid) [inline tests/globalhelper/init.go:12]: Same here, the client seems to be re-created

File: tests/globalhelper/init.go [hunk 3]
```diff
@@ -2,35 +2,43 @@ package globalhelper
...
-		return
+func GetAPIClient() *testclient.ClientSet {
+	if apiclient != nil {
+		return apiclient
 	}
 
-	var err error
-	APIClient, err = config.DefineClients()
-
+	tempClient, err := config.DefineClients()
```
Comment (edcdavid) [inline tests/globalhelper/init.go:21]: why having tempClient and Configuration variable? why not:
var err error
apiclient, err = config.DefineClients()

**Review by edcdavid — APPROVED** [review]

File: tests/globalhelper/init.go (hunk 3 above)
Comment (sebrandon1) [reply tests/globalhelper/init.go:21]: Good call, was a leftover.  I'll update it.

**Pull request merged**